lib/collection/cookie.js

```diff
@@ -271,7 +271,10 @@ _.assign(Cookie.prototype, /** @lends Cookie.prototype */ {
 
         if (this.extensions) {
             this.extensions.forEach(function (ext) {
-                str += '; ' + ext;
+                if (!ext.key || !ext.value) {
+                    return;
+                }
+                str += '; ' + ext.key + '=' + ext.value;
```

Comment:
For `key=` and `httpOnly` value is parsed as `true` so it will be seralized incorrectly here.

Reply:
Incase `key` value is there but `value` is empty we won't set in string due to the `_.isNil` check. We also have unit tests for that, please let me know if I am misunderstanding something here.